sdk-extensions/jaeger-remote-sampler/src/test/java/io/opentelemetry/sdk/extension/trace/jaeger/sampler/JApiCmpTest.java

```diff
@@ -0,0 +1,86 @@
+/*
+ * Copyright The OpenTelemetry Authors
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+package io.opentelemetry.sdk.extension.trace.jaeger.sampler;
+
+import com.google.protobuf.Descriptors;
+import io.opentelemetry.sdk.extension.trace.jaeger.proto.api_v2.Sampling;
+import org.junit.jupiter.api.Disabled;
+import org.junit.jupiter.api.Test;
+
+class JApiCmpTest {
+
+  // TODO: delete before merging
+  @Test
+  @SuppressWarnings({"ModifiedButNotUsed", "ReturnValueIgnored"})
+  @Disabled
+  void compatibility() {
```

Comment:
This test confirms what @anuraaga asserts, which is that the methods have been moved to the parent class.

I've added an exception for this condition to `japicmp`. If folks agree, I'll delete this test before merging, and remove the japicmp exception after the next release.